index.bs

```diff
@@ -341,6 +342,11 @@ The {{height}} attribute MUST return the height in <a lt=px value>CSS pixels</a>
 <a>Picture-in-Picture window</a> associated with {{pictureInPictureElement}} if
 the |state| is |opened|. Otherwise, it MUST return 0.
 
+When the size of the <a>Picture-in-Picture window</a> associated with
+{{pictureInPictureElement}} changes, the user agent MUST <a>queue a task</a> to
+<a>fire an event</a> with the name {{resize}} at {{pictureInPictureElement}}
+with its {{bubbles}} and {{cancelable}} attributes initialized to false.
```

Comment:
no need to mention this at it's already the case by default

Reply:
Done.